src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateClaimSpecCallbackHandler.java

```diff
@@ -369,6 +372,18 @@ private CallbackResponse submitClaim(CallbackParams callbackParams) {
             dataBuilder.respondent2Represented(respondent2Represented);
         }
 
+        if (caseData.getRespondent1Represented() == NO
+            && caseData.getAddRespondent2() == NO
+            && caseData.getAddApplicant2() == NO
+            && toggleService.isPinInPostEnabled()) {
+            dataBuilder.respondent1CaseToPostLRspec(CaseToPostLRspec.builder()
```

Comment:
We should also be populating the expiry date, which would be 6 months from the date of creation

Reply:
Expiry Date generation part is added back in the method